project_timeline/views/project_task_view.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+     Copyright 2017 Tecnativa - Carlos Dauden
+     Copyright 2017 Doscaal - Alexandre Moreau
+     Copyright 2021 Open Source Integrators - Daniel Reis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+    <record id="project_task_timeline" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="type">timeline</field>
+        <field name="arch" type="xml">
+            <timeline
+                date_start="date_start"
+                date_stop="date_end"
+                default_group_by="project_id"
+                event_open_popup="true"
+                colors="white: user_ids == []; #2ecb71: kanban_state == 'done'; #ec7063: kanban_state == 'blocked'"
+            >
```

Comment:
Seeing as V15 now has task dependency built-in, could I suggest we add the following:

```
                colors="white: user_ids == []; #2ecb71: kanban_state == 'done'; #ec7063: kanban_state == 'blocked'"
                dependency_arrow="depend_on_ids"
            >
```

Reply:
Adding what you say, the timeline will look like this:
![imagen](https://user-images.githubusercontent.com/35952655/182784771-f0913a1a-0897-4a30-91c2-2a1696d0b27b.png)

I think the timeline is a bit overloaded

Reply:
Yeah appreciate that, but thought we could deprecate project_task_dependency and project_timeline_task_dependency in favour of they now achieve the same in 15.0?

Reply:
Sorry, I didn't know about the extra modules to add this. Added the feature :smile: